src/app/views/authentication/Authentication.tsx

```diff
@@ -38,7 +38,9 @@ export class Authentication extends Component<IAuthenticationProps, { loginInPro
       this.props.actions!.setQueryResponseStatus({
         ok: false,
         statusText: messages['Authentication failed'],
-        status: errorCode.replace('_', ' '),
+        status: errorCode === 'popup_window_error'
+          ? 'popup blocked, allow pop-up windows in your browser'
+          : errorCode.replace('_', ' '),
```

Comment:
This is neat!! 

We have a localisation file where we deposit the strings so that they are later translated. Please add this `popup blocked, allow pop-up windows in your browser` into [GE.json](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/blob/dev/src/messages/GE.json) . 

Then you can reference it here like so:
```suggestion
        status: errorCode === 'popup_window_error'
          ? translateMessage('popup blocked, allow pop-up windows in your browser')
          : errorCode.replace('_', ' '),
```

Reply:
Alright. I will work on that.